spec/defines/resource_server_spec.rb

```diff
@@ -904,18 +904,6 @@
         it { is_expected.to contain_nginx__resource__location("#{title}-default").with_ssl_only(true) }
       end
 
-      context 'SSL cert missing' do
```

Comment:
Dropping the tests is easier than fixing them, but what's testing that you've correctly chosen the right data types when refactoring?